packages/url/src/clean-for-slug.js

```diff
@@ -23,11 +23,9 @@ export function cleanForSlug( string ) {
 	if ( ! string ) {
 		return '';
 	}
-	return trim(
-		deburr( string )
-			.replace( /[\s\./]+/g, '-' )
-			.replace( /[^\p{L}\p{N}_-]+/gu, '' )
-			.toLowerCase(),
-		'-'
-	);
+	return removeAccents( string )
```

Comment:
Would this work too?

```js
const str = "Crème Brulée"
str.normalize("NFD").replace(/[\u0300-\u036f]/g, "")
> "Creme Brulee"
```

Found it on [StackOverflow](https://stackoverflow.com/questions/990904/remove-accents-diacritics-in-a-string-in-javascript) – perhaps we could do without the `remove-accents` dependency entirely?

Reply:
Thanks for asking @adamziel, it's actually a great question!

I'd be happy to go with that, it's always ideal to not rely on another package. The thing is, that solution works for most of the characters, but from what I've seen, it doesn't cover them all, or at least, doesn't cover as many as the package I've used covers.

Try this in your browser console - I've built it from the character map of the package:

```
"ÀÁÂÃÄÅẤẮẲẴẶÆẦẰȂÇḈÈÉÊËẾḖỀḔḜȆÌÍÎÏḮȊÐÑÒÓÔÕÖØỐṌṒȎÙÚÛÜÝàáâãäåấắẳẵặæầằȃçḉèéêëếḗềḕḝȇìíîïḯȋðñòóôõöøốṍṓȏùúûüýÿĀāĂăĄąĆćĈĉĊċČčC̆c̆ĎďĐđĒēĔĕĖėĘęĚěĜǴĝǵĞğĠġĢģĤĥĦħḪḫĨĩĪīĬĭĮįİıĲĳĴĵĶķḰḱK̆k̆ĹĺĻļĽľĿŀŁłḾḿM̆m̆ŃńŅņŇňŉN̆n̆ŌōŎŏŐőŒœP̆p̆ŔŕŖŗŘřR̆r̆ȒȓŚśŜŝŞȘșşŠšŢţțȚŤťŦŧT̆t̆ŨũŪūŬŭŮůŰűŲųȖȗV̆v̆ŴŵẂẃX̆x̆ŶŷŸY̆y̆ŹźŻżŽžſƒƠơƯưǍǎǏǐǑǒǓǔǕǖǗǘǙǚǛǜỨứṸṹǺǻǼǽǾǿÞþṔṕṤṥX́x́ЃѓЌќA̋a̋E̋e̋I̋i̋ǸǹỒồṐṑỪừẀẁỲỳȀȁȄȅȈȉȌȍȐȑȔȕB̌b̌Č̣č̣Ê̌ê̌F̌f̌ǦǧȞȟJ̌ǰǨǩM̌m̌P̌p̌Q̌q̌Ř̩ř̩ṦṧV̌v̌W̌w̌X̌x̌Y̌y̌A̧a̧B̧b̧ḐḑȨȩƐ̧ɛ̧ḨḩI̧i̧Ɨ̧ɨ̧M̧m̧O̧o̧Q̧q̧U̧u̧X̧x̧Z̧z̧".normalize("NFD").replace(/[\u0300-\u036f]/g, "")
```

and you'll get:

```
AAAAAAAAAAAÆAAACCEEEEEEEEEEIIIIIIÐNOOOOOØOOOOUUUUYaaaaaaaaaaaæaaacceeeeeeeeeeiiiiiiðnoooooøoooouuuuyyAaAaAaCcCcCcCcCcDdĐđEeEeEeEeEeGGggGgGgGgHhĦħHhIiIiIiIiIıĲĳJjKkKkKkLlLlLlĿŀŁłMmMmNnNnNnŉNnOoOoOoŒœPpRrRrRrRrRrSsSsSSssSsTttTTtŦŧTtUuUuUuUuUuUuUuVvWwWwXxYyYYyZzZzZzſƒOoUuAaIiOoUuUuUuUuUuUuUuAaÆæØøÞþPpSsXxГгКкAaEeIiNnOoOoUuWwYyAaEeIiOoRrUuBbCcEeFfGgHhJjKkMmPpQqRrSsVvWwXxYyAaBbDdEeƐɛHhIiƗɨMmOoQqUuXxZz
```

where while the result is satisfactory, you will notice that there are quite a few that actually remain unchanged. 

What do you think?

Reply:
Nice test! 

> For the library above, we also used the corresponding WP function originally when compiling the character map, so this also gets us closer to a parity with the WordPress backend - a nice side benefit IMHO.

Alright, let's go with the package 👍